graylog2-server/src/main/java/org/graylog/security/authservice/rest/AuthServiceBackendsResource.java

```diff
@@ -138,6 +166,55 @@ public void delete(@ApiParam(name = "backendId", required = true) @PathParam("ba
         dbService.delete(config.id());
     }
 
+    @GET
+    @Path("{backendId}/users")
+    @ApiOperation("Get paginated users for active authentication service backend")
+    @RequiresPermissions({RestPermissions.AUTH_SERVICE_GLOBAL_CONFIG_READ, RestPermissions.USERS_LIST})
```

Comment:
I'd prefer it if we used `USERS_READ`. 
Or even `GranteeService.getVisibleUsers()`

Reply:
I opted for `USERS_READ`. I think we need to have a larger discussion about fine grained permission checks. The UI somehow needs to handle this as well. We probably need placeholder user objects if the requesting user is not allowed to read a user. Otherwise there would be no indicator that there are users for that backend.